elliottlib/cli/__main__.py

```diff
@@ -362,6 +378,11 @@ def repair_bugs(runtime, advisory, auto, id, original_state, new_state, noop, de
         if bug.status in original_state:
             changed_bug_count += 1
             elliottlib.bzutil.set_state(bug, new_state, noop=noop)
+        if close_placeholder and "Placeholder" in bug.summary:
+            changed_bug_count += 1
+            elliottlib.bzutil.set_state(bug, "CLOSED", noop=noop)
```

Comment:
This would move placeholder bugs even if it is not in original_state.
Also for placeholder bugs, `changed_bug_count` may be added twice, and the state may be unnecessarily changed twice.
```suggestion
        if bug.status in original_state:
            if close_placeholder and "Placeholder" in bug.summary:
                elliottlib.bzutil.set_state(bug, "CLOSED", noop=noop)
            else:
                elliottlib.bzutil.set_state(bug, new_state, noop=noop)
            changed_bug_count += 1
```